var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -140,14 +151,10 @@ def mpi_dependent_options(self):
                 raise RuntimeError('\n'.join(errors))
         else:
             compiler_opts = [
-                '--with-mpi=1',
```

Comment:
does `petsc` not need this to understand that the provided compilers are MPI wrappers?